src/compiler/declarationEmitter.ts

```diff
@@ -913,6 +916,10 @@ namespace ts {
 
             emitJsDocComments(node);
             emitModuleElementDeclarationFlags(node);
+            if (node.flags & NodeFlags.Abstract) {

```

Comment:
In principle, I see no reason to disallow default abstract classes. Why are they any different from concrete classes from an export default standpoint?